portfolio/src/main/webapp/script.js

```diff
@@ -87,11 +97,11 @@ function createListElement(comment) {
   timestampElement.setAttribute('class', 'time');
   messageElement.setAttribute('class', 'msg');
 
-  if (typeof(comment.userEmail) === "undefined") {
+  if (typeof(comment.user) === "undefined") {
```

Comment:
It seems to me you always return comment.user wether a DisplayName is set or not. If so, then you don't need to check typeof(comment.user) === "undefined", am I understanding it correctly?

Reply:
Ah this one is because previously in last week, I did the comment part and at that time I did not require for the user's email or display name, so for those saved comments, I just call the users as 'anonymous user'

Reply:
I see. Then you can remove this piece of code now. We constantly keep only useful code, so that codebase doesn't rot too soon.

Reply:
Okay, sure!